storybook/preview.js

```diff
@@ -27,4 +27,16 @@ export const parameters = {
 		// Will be enabled on a per-story basis until migration is complete.
 		disable: true,
 	},
+	options: {
+		storySort: {
+			order: [
+				'Docs',
+				'Playground',
+				'BlockEditor',
+				'Components',
+				'Components (Experimental)',
+				'Icons',
+			],
```

Comment:
What happens if we add another section to Storybook and we forget to add it to this sort list? Will it get displayed anyway or will it stay hidden?

Reply:
Good question. Unlisted sections will just be displayed after all the listed ones. And if someone is unhappy with that display order, a google search for "storybook section order" or something should easily guide their way to this config file 👍